include/ebpf_store_helper.h

```diff
@@ -349,3 +350,49 @@ ebpf_store_update_global_helper_information(
 
     return status;
 }
+
+static __return_type
+ebpf_store_clear(_In_ const ebpf_registry_key_t root_key_path)
```

Comment:
As per the coding guidelines, static functions should begin with _.

Reply:
@Alan-Jowett got it. Just heads up I here mimicked what done already with:

- `ebpf_store_update_section_information`
- `ebpf_store_update_program_information`
- `ebpf_store_update_global_helper_information`

Should we change these to `_*` as well?

Reply:
> @Alan-Jowett got it. Just heads up I here mimicked what done already with:
> 
> * `ebpf_store_update_section_information`
> * `ebpf_store_update_program_information`
> * `ebpf_store_update_global_helper_information`
> 
> Should we change these to `_*` as well?

IMO, _new_ code ought to be coding guidelines compliant.  For the other (existing) instances such as those mentioned above, maybe we could file a new issue to address those?

Reply:
No problem then, they are few enough to be fixed as per coding guidelines.

Reply:
@saxena-anurag for comment as per internal discussion.